### DISCUSSION TITLE
Coverage Map

### DISCUSSION THREAD

**Discussion by nourhan0106** [NVlabs/sionna#537] [Q&A] [answered]
I know from the documentation that the coverage map represents the path gain and to generate a coverage map the represents the SINR we should scale it. However, I am not sure how could we scale it and how is the path gain calculated is it the sum of rays received at each point?

**Comment by akdd11** [comment, marked as answer]
Hi @nourhan0106 ,

from the coverage map and a given transmit power you can obtain the signal level. For interference and noise, you either have to make some assumptions on a fixed value (e.g. considering thermal noise for your receiver) or if you want to consider another transmitter active in the same band creating interference, you can create an coverage map for the other transmitter and consider the obtained signal level as interference for the initial transmitter.

**Comment by jhoydis** [comment]
Hi @nourhan0106,

Adding to @akdd11's response, you can find a definition of the coverage map [here](https://nvlabs.github.io/sionna/api/rt.html#coverage-map). A coverage map is independent of the transmit and noise power in a system and just describes the pathloss over a certain area. It is an approximation of $\sum_i |a_i|^2$ with $a_i$ defined in [Eq.(26)](https://nvlabs.github.io/sionna/em_primer.html#equation-h-final).

**Reply by nourhan0106** [reply]
I am trying to compute the a_i from cir() function with 1 tx with 1 antenna and 1 rx with 1 antenna and it gives me a list of shape 
(1,1,2,1,2,35,1)
which corresponds to (batch num, tx num, num_tx_ant, num_rx, num_rx_ant, max_num_paths, num_time_steps)
but I am not sure where does 2 comes from if I am using num_tx_ant=num_rx_ant=1

**Reply by jhoydis** [reply]
I think that you might be using dual-polarized antennas. Each polarization direction corresponds to a separate antenna.

**Comment by lufeiaixuexi** [comment]
Hi,@nourhan0106,
May I ask if your question has been answered? I also want to know how to generate a coverage map the represents the SINR?
I saw an idea in the discussion forum,
Compute SINR:
Signal Power (S): Calculate the received signal power from the desired transmitter at the receiver’s position.
Interference Power (I): Calculate the sum of the received power from all other interfering transmitters.
Noise Power (N): Define the noise power based on the system’s noise figure.
The SINR can be computed as:
SINR=S/(I+N)
But I don't know how to calculate N and I.
Thank you very much！

**Reply by nourhan0106** [reply]
I am not sure but I think for the noise you could assume a gaussian distribution with avg noise power from a practical scenario and for the Interference maybe you should compute it from cir function of different transmitters and add it

**Comment by faycalaa** [comment]
Hi,

The next release of Sionna will include a feature for computing SINR maps, which should help address your issue.

**Reply by lufeiaixuexi** [reply]
Hello, may I ask when the new version of Sionna will be released?
